greptimedb/load_data.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Check if the required arguments are provided
+if [[ $# -lt 4 ]]; then
+    echo "Usage: $0 <DATA_DIRECTORY> <MAX_FILES> <SUCCESS_LOG> <ERROR_LOG>"
+    exit 1
+fi
+
+# Arguments
+DATA_DIRECTORY="$1"
+MAX_FILES="$2"
+SUCCESS_LOG="$3"
+ERROR_LOG="$4"
+
+# Validate arguments
+[[ ! -d "$DATA_DIRECTORY" ]] && { echo "Error: Data directory '$DATA_DIRECTORY' does not exist."; exit 1; }
+[[ ! "$MAX_FILES" =~ ^[0-9]+$ ]] && { echo "Error: MAX_FILES must be a positive integer."; exit 1; }
+
+pushd $DATA_DIRECTORY
+counter=0
+for file in $(ls *.json.gz | head -n $MAX_FILES); do
+    echo "Processing file: $file"
+
+    curl "http://localhost:4000/v1/events/logs?table=jsontable&pipeline_name=jsonbench&ignore_errors=true" \
```

Comment:
I guess calling the table "jsonbench" will be slightly less generic.

Reply:
Change the table name to `bluesky` here. `jsonbench` is the pipeline config's name.